### ISSUE TITLE
server crash (AE2WTLib-1.2.3-beta.5 + appliedenergistics2-fabric-8.3.0-alpha.3)

### ISSUE THREAD

**Issue by MSandro** [Mari023/AE2WirelessTerminalLibrary#71]
no wireless terminals used yet in the world but it crashed rarely but very spontaneously and unpredictably.
```
---- Minecraft Crash Report ----
// I'm sorry, Dave.

Time: 7/14/21, 11:56 AM
Description: Exception in server tick loop

java.lang.NoClassDefFoundError: net/minecraft/class_746
	at de.mari_023.fabric.ae2wtlib.wct.CraftingTerminalHandler.getCraftingTerminalHandler(CraftingTerminalHandler.java:55)
	at de.mari_023.fabric.ae2wtlib.wct.magnet_card.MagnetHandler.doMagnet(MagnetHandler.java:28)
	at net.fabricmc.fabric.api.event.lifecycle.v1.ServerTickEvents.lambda$static$0(ServerTickEvents.java:40)
	at net.minecraft.server.MinecraftServer.handler$ccj000$onStartTick(MinecraftServer.java:5258)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:811)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:670)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_746
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at net.fabricmc.loader.launch.server.InjectingURLClassLoader.loadClass(InjectingURLClassLoader.java:56)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:168)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 8 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Linux (amd64) version 4.4.0-154-generic
	Java Version: 11.0.9.1, Oracle Corporation
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1581728648 bytes (1508 MB) / 4060086272 bytes (3872 MB) up to 6291456000 bytes (6000 MB)
	CPUs: 3
	JVM Flags: 18 total; -Xmx6000M -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1
	Fabric Mods: 
		addendum: Addendum 1.16.5-1.0.2
		adorn: Adorn 1.14.1+1.16.5
		advanced_runtime_resource_pack: Runtime Resource Pack 0.2.9
		advdebug: Advancements Debug 2.3.0
		ae2wtlib: AE2WTLib 1.2.3-beta.5
		agape_space: Agape Space Mod 0.0.16
		alloy_forgery: Alloy Forgery 1.2.0
		aopalliance_aopalliance: aopalliance 1.0
		appleskin: AppleSkin mc1.16.4-2.0.0
		appliedenergistics2: Applied Energistics 2 8.3.0-alpha.3
		architectury: Architectury 1.19.27
		artifice: Artifice 0.14.6+1.16.5
		attributefix: Attribute Fix {FABRIC} 1.0.1
		auto_planter: AutoPlanter 16.1-fabric
		autoconfig1u: Auto Config v1 Updated 3.3.1
		autoconfigtoml: AutoConfig-TOML 1.0.0-fabric
		bclib: BCLib 0.1.45
		betterend: Better End 0.9.8.5-pre
		blockshifter: Blockshifter 1.3.2+fabric-1.16.4
		blockus: Blockus 2.1.9+1.16.5
		blue_endless_jankson: jankson 1.2.0
		campanion: Campanion 1.3.5
		cardinal-components: Cardinal Components API 2.8.2
		cardinal-components-base: Cardinal Components API (base) 2.8.3
		cardinal-components-block: Cardinal Components API (blocks) 2.8.2
		cardinal-components-chunk: Cardinal Components API (chunks) 2.8.2
		cardinal-components-entity: Cardinal Components API (entities) 2.8.3
		cardinal-components-item: Cardinal Components API (items) 2.8.2
		cardinal-components-level: Cardinal Components API (world saves) 2.8.2
		cardinal-components-scoreboard: Cardinal Components API (scoreboard) 2.8.2
		cardinal-components-util: Cardinal Components API (utilities) 2.8.3
		cardinal-components-world: Cardinal Components API (worlds) 2.8.2
		carrier: Carrier 1.7.6
		chainmail: Chainmail 0.5.0
		charm: Charm 2.3.2
		cloth-api: Cloth API 1.4.9
		cloth-armor-api-v1: Cloth Armor API v1 1.5.47
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-client-events-v0: Cloth Client Events v0 1.5.47
		cloth-common-events-v1: Cloth Common Events v1 1.5.47
		cloth-config2: Cloth Config v4 4.11.26
		cloth-datagen-api-v1: Cloth Datagen v1 1.4.9
		cloth-durability-bar-api-v1: Cloth Durability Bar API v1 1.5.47
		cloth-dynamic-registry-api-v1: Cloth Dynamic Registry API v1 1.4.9
		cloth-utils-v1: Cloth Utils v1 1.4.9
		com_electronwill_night-config_core: core 3.6.3
		com_electronwill_night-config_toml: toml 3.6.3
		com_fasterxml_jackson_core_jackson-annotations: jackson-annotations 2.12.3
		com_fasterxml_jackson_core_jackson-core: jackson-core 2.12.3
		com_fasterxml_jackson_core_jackson-databind: jackson-databind 2.12.3
		com_fasterxml_jackson_dataformat_jackson-dataformat-yaml: jackson-dataformat-yaml 2.12.3
		com_fasterxml_jackson_module_jackson-module-kotlin: jackson-module-kotlin 2.12.3
		com_github_nathanpb_safer: Safer 1.1.0
		com_google_inject_guice: guice 4.2.3
		com_moandjiezana_toml_toml4j: toml4j 0.7.2
		completeconfig: CompleteConfig 0.17.1
		compostable-rottenflesh: Compostable Rotten Flesh 1.0.0
		compressedblocks: Compressed Blocks 1.0
		computercraft: CC:Restitched 1.96.1-rc1
		couplings: Couplings 1.3.3
		crookedcrooks: Crooked Crooks 1.3.3+1.16.4
		croptopia: Croptopia 1.4.0
		croptosis: Croptosis 1.1.0
		crowdin-translate: CrowdinTranslate 1.3+1.16
		curios: Curios API 0.0.13-1.16.5
		customportalapi: Custom Portals 0.0.1-beta32-1.16
		dankstorage: Dank Storage 1.9a-1.16.3
		darkpaintings: DarkPaintings 3.0.2
		datapackportals: Datapack Portals 1.0.2-1.16
		diggusmaximus: Diggus Maximus 1.4.2-1.16.3
		dml-refabricated: Deep Mob Learning: Refabricated 0.3.3-beta1
		dynamo_api: Dynamo API 1.0.0+1.16
		easyautocrafting: EasyAutoCrafting 1.1+1.16
		engilib: EngiLib 1.0.0+1.16.4
		exotic-art-core: Exotic Art Core Content Pack 1.0.379
		exotic-matter: Exotic Matter 2.12.381
		expandedstorage: ExpandedStorage 5.6.33
		extraalchemy: Extra Alchemy 1.4.5
		extragenerators: Extra Generators 1.0.3-BETA+1.16
		fabric: Fabric API 0.35.0+1.16
		fabric-api-base: Fabric API Base 0.3.0+c88702897d
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.2.0+84310a227d
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.10+3661eaa17d
		fabric-command-api-v1: Fabric Command API (v1) 1.1.1+351679a77d
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+ca58154a7d
		fabric-containers-v0: Fabric Containers (v0) 0.1.11+9354966b7d
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.1+ca58154a7d
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+ca58154a7d
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.0.6+61336ba97d
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.1.0+c88702897d
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.2+ca58154a7d
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+ca58154a7d
		fabric-furnaces: Fabric Furnaces 1.3.3-1.16.4
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+ca58154a7d
		fabric-item-api-v1: Fabric Item API (v1) 1.2.1+ca58154a7d
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.7+2868a2287d
		fabric-language-kotlin: Fabric Language Kotlin 1.6.1+kotlin.1.5.10
		fabric-language-scala: Fabric Language Scala 1.0.0
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.1+ca58154a7d
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.2+ca58154a7d
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+ca58154a7d
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.3+e3c9d0627d
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.8+ca58154a7d
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+ca58154a7d
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.5+c88702897d
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.2+ca58154a7d
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+ca58154a7d
		fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.4+ca58154a7d
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+ca58154a7d
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.6+2868a2287d
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.5+9354966b7d
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.4+ca58154a7d
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.1+ca58154a7d
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.7+8183d3957d
		fabric-transfer-api-v1: Fabric Transfer API (v1) 0.2.2+3185237c7d
		fabricloader: Fabric Loader 0.11.3
		fallflyinglib: FallFlyingLib 2.0.4
		fasttransferlib: Fast Transfer Lib 0.4.2
		fatxporbs: Fat Experience Orbs 0.0.9
		fermion: Fermion 2.9.218
		fermion-modkeys: Fermion Modifier Keys 1.10.218
		fermion-orientation: Fermion Orientation 1.2.218
		fermion-varia: Fermion Varia 2.6.218
		fiber: fiber 0.23.0-2
		ftbchunks: FTB Chunks 1605.3.1-build.24
		ftblibrary: FTB Library 1605.3.1-build.48
		ftbteams: FTB Teams 1605.2.1-build.17
		go-fish: Go Fish 1.2.3-beta-1.16.5
		golemsgalore: Golems Galore 0.2.1+mc.1.16.5
		gunpowder-base: Gunpowder 0.5.7+1.16.5
		gunpowder-sleepvote: Gunpowder SleepVote 1.0.4+gunpowder.0.5.4.mc.1.16.5
		gunpowder-teleport: Gunpowder Teleport 1.0.6+gunpowder.0.5.2.mc.1.16.5
		gunpowder-utilities: Gunpowder Utilities 1.0.4+gunpowder.0.4.1.mc.1.16.5
		harvest_scythes: Harvest Scythes 2.0.4
		hctm-base: HCTM Base 3.5.1
		improved-stations: Improved Stations 2.0.3
		indrev: Industrial Revolution 1.9.18-BETA
		inmis: Inmis 2.2.1-1.16.5
		inventorysorter: Inventory Sorter 1.7.9-1.16
		iron-jetpacks: Iron Jetpacks 0.3.1
		jankson: Jankson 3.0.1+j1.2.0
		java: OpenJDK 64-Bit Server VM 11
		javax_inject_javax_inject: javax.inject 1
		kibe: Kibe 1.7-BETA
		ktdatataglib: KtDataTagLib 1.4.0
		kubejs: KubeJS 1605.3.11-build.35
		kyrptconfig: Kytpt Config 1.1.6-1.16
		lapisreserve: Lapis Reserve 1.0.7
		libblockattributes: LibBlockAttributes 0.8.8
		libblockattributes_core: LibBlockAttributes (Core) 0.8.8
		libblockattributes_fluids: LibBlockAttributes (Fluids) 0.8.8
		libblockattributes_items: LibBlockAttributes (Items) 0.8.8
		libcd: LibCapableData 3.0.3+1.16.3
		libgui: LibGui 3.4.0+1.16.5
		libmultipart: LibMultiPart 0.5.0
		libnetworkstack: Lib Network Stack 0.4.7
		lithium: Lithium 0.6.6
		magna: Magna 1.5.1-1.16.5
		mcdict: MCDict 2.0.1+1.16.4
		mechanicaltech: Mechanical Tech 0.3.0+1.16
		megane: megane 4.1.3
		megane-applied-energistics-2: megane-applied-energistics-2 4.1.3+8.3.0-alpha.1
		megane-base: megane-base 4.1.3
		megane-fabric-furnaces: megane-fabric-furnaces 4.1.3+1.3.3-1.16.4
		megane-fast-transfer-lib: megane-fast-transfer-lib 4.1.3+0.3.1
		megane-industrial-revolution: megane-industrial-revolution 4.1.3+1.8.13-BETA
		megane-kibe: megane-kibe 4.1.3+1.6.6-BETA
		megane-lib-block-attributes: megane-lib-block-attributes 4.1.3+0.8.7
		megane-modern-industrialization-4: megane-modern-industrialization-4 4.1.3+0.4.0-alpha.07
		megane-runtime: megane-runtime 4.1.3
		megane-team-reborn-energy: megane-team-reborn-energy 4.1.3+0.1.0
		megane-tech-reborn: megane-tech-reborn 4.1.3+3.8.2-build.220
		megane-vanilla: megane-vanilla 4.1.3+1.16.5
		microconfig: Microconfig 1.0.5
		minecraft: Minecraft 1.16.5
		mo_glass: Mo Glass 1.5-MC1.16.5
		modern_industrialization: Modern Industrialization 0.4.0-alpha.10
		more_codecs: More Codecs 0.1.2
		mysql_mysql-connector-java: mysql-connector-java 5.1.46
		mysticalagriculture: Mystical Agriculture: Refabricated 1.1.7
		oldfishfarm: Old Fish Farm 1.0.0 1.16.1
		omega-config: OmegaConfig 1.0.2
		org_jetbrains_exposed_exposed-core: exposed-core 0.31.1
		org_jetbrains_exposed_exposed-java-time: exposed-java-time 0.31.1
		org_jetbrains_exposed_exposed-jdbc: exposed-jdbc 0.31.1
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.5.10
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.5.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.5.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.5.10
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.5.0
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.5.0
		org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.2.1
		org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.2.1
		org_objenesis_objenesis: objenesis 3.2
		org_postgresql_postgresql: postgresql 42.2.2
		org_slf4j_slf4j-api: slf4j-api 1.8.0-beta1
		org_xerial_sqlite-jdbc: sqlite-jdbc 3.28.0
		org_yaml_snakeyaml: snakeyaml 1.26
		patchouli: Patchouli 1.16.4-53-FABRIC
		pehkui: Pehkui 2.2.1+1.14.4-1.17
		phosphor: Phosphor 0.7.2+build.12
		plantinajar: Plant In A Jar 2.1.2
		player_roles: Player Roles 1.3.1
		playerabilitylib: Pal 1.2.2
		qcommon-cfg-core: qcommon-cfg-core 3.2.0
		qcommon-croco: Croco 2.1.1
		reach-entity-attributes: Reach Entity Attributes 1.1.1
		reborncore: Reborn Core 4.7.3+build.136
		rhino: Rhino 1605.1.1-build.23
		roughlyenoughitems: Roughly Enough Items 5.12.248
		roughlyenoughitems-api: REI (API) 5.12.248
		roughlyenoughitems-default-plugin: REI (Default Plugin) 5.12.248
		roughlyenoughitems-runtime: REI (Runtime) 5.12.248
		roughlyenoughresources: Roughly Enough Resources 2.1.5
		rswires: RSWires 4.10.2
		scf: Super Cobblestone Funnels 1.2
		schmucks: Schmucks 1.3.2
		seasons: Fabric Seasons 1.2-BETA
		secretrooms: Secret Rooms 0.9.5+1.16.5
		seedy-behavior: Seedy Behavior 1.0.1
		selfgoverningcatalyst: Self-Governing Catalyst 1.16-1.0.6
		simple_pipes: SimplePipeTest 0.5.0
		simple_pipes_dep_container: SimplePipeTest (Dependency Container) 0.5.0
		skyutils: Sky utilities Mod 1.0.2
		slabhelper: Slab Helper 1.3.4
		slabstoblocks: SlabsToBlocks 1.0.0
		someforgepatchesported: Some Forge Patches Ported 1.0.0
		spark: spark 1.6.0
		spatialharvesters: Spatial Harvesters 0.11.10
		special-circumstances: Special Circumstances 1.8.218
		staffofbuilding: Staff of Building 1.2.6-1.16.5
		starry_sky: Starry Skies 1.0.10-1.16.5
		step-height-entity-attribute: Step Height Entity Attribute 1.0.0
		structurized: Structurized 1.2.0-1.16.5
		tatters: Tatters 0.0.3+1.16
		team_reborn_energy: Energy 0.1.1
		techreborn: Tech Reborn 3.8.4+build.236
		terrarianslimes: Terrarian Slimes 1.1-BETA+1.16
		the_aether: The Aether 1.16.5-1.4.1
		things: Things 0.1.3
		timeinabottle: Time in a Bottle 1.3-mc1.16.4-fabric
		toms_storage: Tom's Simple Storage Mod 1.1.12
		totemmod: Wither Totems Mod 0.3.2
		trapexpansion: Trap Expansion 1.3.2
		trinkets: Trinkets 2.6.7
		truerooms: True Secret Rooms 1.1.3
		unlimiteddragoneggs: Unlimited Dragon Eggs 1.1.1
		vanilladeathchest: VanillaDeathChest 2.0.6-fabric
		winged: Winged Mod 1.16.5-2.1.4
		wirelessnetworks: Wireless Networks 1.0.3
		woodenhoppers: Wooden Hoppers 1.0.0
		wthit: wthit 2.10.4
		xb: Exotic Blocks 1.6.172
	Player Count: 1 / 20; [class_3222['MSandro'/466, l='ServerLevel[world]', x=7.50, y=81.00, z=10.50]]
	Data Packs: vanilla (incompatible), Fabric Mods, secretrooms:server_pack
	Is Modded: Definitely; Server brand changed to 'Fabric/Gunpowder'
	Type: Dedicated Server (map_server.txt)
```

**Comment by Mari023** [comment]
class_746 is ClientPlayerEntity (a client only class)
I'll look into it and hopefully have a fixed version soon

**Comment by Mari023** [comment]
fixed with 1.2.3-beta.6 (released soon)
